intranet/apps/dataimport/management/commands/import_eighth.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from intranet.apps.eighth.models import EighthActivity, EighthRoom, EighthSponsor
+
+
+class Command(BaseCommand):
+    help = "Import Fake Activities"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("data_fname")
+
+    def handle(self, *args, **kwargs):
+        try:
+            with open(kwargs["data_fname"]) as f_obj:
+                data = json.load(f_obj)
+        except OSError as ex:
+            raise CommandError(str(ex)) from ex
+
+        for activity in data["activities"]:
+            name = activity["Name"].strip()
+            description = activity["Description"].strip()
+            sponsor_name = activity["Sponsor"].strip().split()
```

Comment:
The later splicing of `sponsor_name` is confusing. Please instead split it into `sponsor_first_name` and `sponsor_last_name` and use these instead of `sponsor_name[0]` and `sponsor_name[1]`.